cmd/jujud/main.go

```diff
@@ -24,25 +24,25 @@ import (
 	proxyutils "github.com/juju/proxy"
 	"github.com/juju/utils/exec"
 
+	k8sexec "github.com/juju/juju/caas/kubernetes/provider/exec"
 	jujucmd "github.com/juju/juju/cmd"
 	agentcmd "github.com/juju/juju/cmd/jujud/agent"
 	"github.com/juju/juju/cmd/jujud/agent/caasoperator"
 	"github.com/juju/juju/cmd/jujud/dumplogs"
 	"github.com/juju/juju/cmd/jujud/introspect"
+	"github.com/juju/juju/cmd/jujud/run"
 	cmdutil "github.com/juju/juju/cmd/jujud/util"
 	components "github.com/juju/juju/component/all"
 	"github.com/juju/juju/core/machinelock"
 	jujunames "github.com/juju/juju/juju/names"
 	"github.com/juju/juju/juju/sockets"
-
-	k8sexec "github.com/juju/juju/caas/kubernetes/provider/exec"
-
-	// Import the providers.
-	_ "github.com/juju/juju/provider/all"
 	"github.com/juju/juju/upgrades"
 	"github.com/juju/juju/utils/proxy"
 	"github.com/juju/juju/worker/logsender"
 	"github.com/juju/juju/worker/uniter/runner/jujuc"
+
+	// Import the providers.
+	_ "github.com/juju/juju/provider/all"
```

Comment:
I don't think we need this. All of the provider specific code should be in the apiserver.

This may bring in a bunch of things we don't want.

Reply:
Sorry, I see now that this is jujud main. This is fine.